webextensions/api/extension.json

```diff
@@ -174,7 +174,7 @@
             "__compat": {
               "support": {
                 "chrome": {
-                  "version_added": "≤93"
+                  "version_added": "5"
```

Comment:
✅ https://github.com/chromium/chromium/commit/37ad8892fe9576eef898295e50e7d42b5d7a3e27 in [5.0.364.0](https://github.com/chromium/chromium/releases/tag/5.0.364.0) before 5.0.375.